shaded/distributedlog-core-shaded/pom.xml

```diff
@@ -82,8 +82,8 @@
                   <include>com.fasterxml.jackson.core:jackson-annotations</include>
                   <include>com.google.guava:guava</include>
                   <include>com.google.protobuf:protobuf-java</include>
-                  <!-- netty 3 (included from zookeeper) -->
-                  <include>io.netty:netty</include>
+                  <!-- netty (included from zookeeper) -->
+                  <include>io.netty:netty-all</include>
```

Comment:
this is strage.
we did not shade netty 4 before, only netty 3, that is a different library